README.md

```diff
@@ -145,7 +145,7 @@ for domain in dumped_json['domains']:
     print(f"{domain} has CDNs:\n {dumped_json['domains'][domain]['cdns']}")
 ```
 
-## How Does it Work
+## How Does it Work ##
```

Comment:
I suggest adding a question mark to the end of this heading.

Reply:
This is against `markdownlint` rule `MD026`, please see https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md026---trailing-punctuation-in-heading

Reply:
Perhaps instead this heading can be changed so it is not a question, e.g. "How It Works".

Reply:
I like the change @dav3r mentioned, but I'm also OK with leaving it as is.

Reply:
Thanks for the suggestion @dav3r , updated in 6adec78.